dotCMS/src/integration-test/java/com/dotcms/storage/FileMetadataAPITest.java

```diff
@@ -156,35 +259,36 @@ private void validateFull(final Map<String, Serializable> metaData, final TestFi
      * validate basic layout expected in the basic md for File-Asset
      * @param metaData
      */
-    private void validateBasic(final Map<String, Serializable> metaData){
+    private void validateBasic(final Metadata metaData){
+        final Map<String, Serializable> meta = metaData.getFieldsMeta();
         basicMetadataFields.forEach(key -> {
-            assertTrue(metaData.containsKey(key));
+
+            if (key.equals("width") || key.equals("height") && !(Boolean)meta.get("isImage")) {
```

Comment:
I think this validation should be `((key.equals("width") || key.equals("height")) && !(Boolean)meta.get("isImage"))`

Reply:
or to make it more clear: `((Boolean)meta.get("isImage") && !(key.equals("width") || key.equals("height")))`